ci: change issue template order

rickstaa opened the pull request:
**What does this pull request do? Explain your changes. (required)**

This commit ensures that the main branch issue templates are put above the AI related issue templates. Sadly this change can not yet be performed through the [GitHub settings](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#changing-the-order-of-templates). As a result we need to prefix the issue templates.

**Specific updates (required)**

- Issue template names changed to change order.

**How did you test each of these updates (required)**

<!-- A detailed description of how you tested your code changes. Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code, etc. -->

Not needed.

**Does this pull request close any open issues?**
<!-- Fixes # -->

No

**Checklist:**
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] Read the [contribution guide](./CONTRIBUTING.md)
- [x] `make` runs successfully
- [x] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [ ] [Pending changelog](./CHANGELOG_PENDING.md) updated

rickstaa committed:
ci: change issue template order

This commit ensures that the main branch issue templates are put above
the AI related issue templates.

hjpotter92 reviewed (approved):
you can rename the ai feature specific template files to a higher index, for eg. `50-ai-bug-report.md`

rickstaa committed:
ci(ai): add PR labeler config file

This commmit adds a https://github.com/actions/labeler configuration
file so that all PRs on the `ai-video` branch will be correctly labeled
with the `ai` label.
